projects/mi_sensitive_human_data/MIROPA/schema/data_schema.json

```diff
@@ -0,0 +1,87 @@
+{
+    "$id": "http://some-github-repo.url/schemas/data",
+    "$schema": "http://json-schema.org/draft-07/schema",
+    "title": "MIROPA schema for Data entity",
+    "description": " Sensitive human data as defined by GDPR",
+    "type": "object",
+    "properties": {
+        "@context": {
+            "description": "The JSON-LD context",
+            "anyOf": [
+                {
+                      "type": "string"
+                },
+                {
+                      "type": "object"
+                },
+                {
+                      "type": "array"
+                }
+                  ]
+          },
+          "@id": {
+            "description": "The JSON-LD identifier",
+            "type": "string",
+            "format": "uri" 
+          },
+          "@type": {
+            "description": "The JSON-LD type",
+            "type": "string",
+            "const": "Data"
+          },
+      
+      
+      "category_of_personal_data": {
+        "description": "  Any information which are related to an identified or identifiable natural person (Art. 4 (1) GDPR) .",
+        "type": "array",
+        "items" : {
+         "$ref": "person_schema.json#"
```

Comment:
This I believe is not correct. `person_schema.json` defines person object, not category of personal data. Could you change this property to array with controlled list of values? Something like [here](https://github.com/elixir-luxembourg/json-schemas/blob/d8a8fd818cfd0997c7202485a7884e46e251613d/schemas/elu-dataset.json#L89)

Reply:
@vildead , I wanted to separate the categories of personal data into two: the standard data : name, surname, address, (maybe  I can rename it maybe to standard personal data) and the special data (Genetic, biometric, etc). That's why I used the file person_shema.json# ras ref. or  I can group everything together and include the standard information.